Makefile

```diff
@@ -38,6 +38,5 @@ upgrade: $(COMMON_CONSTRAINTS_TXT)  ## update the requirements/*.txt files with
 	pip-compile --upgrade -o requirements/development.txt requirements/development.in
 	for fextra in edx_repo_tools/*/extra.in; do pip-compile --upgrade -o $${fextra%.in}.txt $$fextra; done
 
-lint:				## run pep8 and pylint
-	pep8 || true
+lint:				## run pylint
 	pylint *.py edx_repo_tools tests || true
```

Comment:
What's with the "Or True" here, should that be removed?

Reply:
Good point, it was being used to let targets succeed and ignore the exit status. It was important on pep8 so that pylint would also run, but we don't need it here.